specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/examples/AvailableWorkloadProfiles_Get.json

```diff
@@ -124,6 +124,19 @@
             "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/providers/Microsoft.App/availableManagedEnvironmentsWorkloadProfileTypes/CPUOptimized-3",
             "name": "CPUOptimized-3",
             "type": "Microsoft.App/availableManagedEnvironmentsWorkloadProfileTypes"
+          },
+          {
+            "location": "East US",
+            "properties": {
```

Comment:
Update display names to match https://msazure.visualstudio.com/Antares/_workitems/edit/17304240